main.go

```diff
@@ -126,6 +128,18 @@ func main() {
 		setupLog.Error(err, "unable to create controller", "controller", "Supervision")
 		os.Exit(1)
 	}
+	if err = (&controllers.PriorityExpanderReconciler{
+		Client: mgr.GetClient(),
+		Scheme: mgr.GetScheme(),
+		Configuration: controllers.PriorityExpanderReconcilerConfiguration{
+			ClusterAutoscalerStatusNamespace: clusterAutoscalerStatusNamespace,
+			ClusterAutoscalerStatusName:      clusterAutoscalerStatusName,
+			ClusterAutoscalerPEConfigMapName: clusterAutoscalerPEConfigMapName,
+		},
+	}).SetupWithManager(mgr); err != nil {
+		setupLog.Error(err, "unable to create controller", "controller", "PriorityExpander")
+		os.Exit(1)
+	}
```

Comment:
You should add a flag with the `PriorityExpander` to reconcile and instantiate the controller only if that flag is set.